docs/UserGuide.md

```diff
@@ -163,9 +167,11 @@ Edits the specified task from the planner.
 
 #### Format
 `edit INDEX [n/NAME] [t/TIME] [d/DURATION] [D/DEADLINE] [r/RECURRENCE] [i/IMPORTANCE] [a/ADDITIONAL-NOTES]`
+`e INDEX [n/NAME] [t/TIME] [d/DURATION] [D/DEADLINE] [r/RECURRENCE] [i/IMPORTANCE] [a/ADDITIONAL-NOTES]`
+* At least one parameter have to be edited.
 
 #### Parameters
-* `INDEX` refers to the index number shown in the displayed task list. It must be a positive integer.
+* `INDEX` refers to the index number shown in the displayed task list. It cannot be a negative integer.
```

Comment:
Perhaps say "It must be a non-negative integer". Also we should change`INDEX` to `Task ID` soon

Reply:
Yea I think we shouldn't say "it cannot be a negative integer", because it also cannot be many other things. So better to say what it "must be".